config/initializers/scheduler.rb

```diff
@@ -11,9 +11,6 @@
   #puts "calculating recommendation counts..."
   #Person.recalc_recommendation_counts
 end
-scheduler.every '36h' do
-  Manifestation.get_popular_works
```

Comment:
I believe the purpose of this was to prepopulate Popular works cache by schedule, but I think it is redundant.